script/Arcueid/Day 5/05_00_ARC05_1B.txt

```diff
@@ -698,17 +700,17 @@ A person who kills for no reason should never have had a place in society to beg
 [sha:7ef5ed2bea3a675f5f69444c0518a3884a579527]{
 -- Page 94, Offset 8812.
 -- 志貴は本当に、自分でも理由が分からないのね」
-You really don't understand the reason yourself, do you?"
+You really don't understand why you did it yourself, do you?"
```

Comment:
drop "yourself", with this structure it sounds like "without help" rather than introspectively